rails-tasks/app/controllers/tasks_controller.rb

```diff
@@ -21,7 +20,24 @@ def show
   end
 
   def new
+    @task = Task.new
+  end
+
+  def create
+    Task.create(task_params[:task])
+    redirect_to "/"
+  end
+
+  def destroy
+    to_delete = Task.where(task_params[:name])
```

Comment:
Having your routes use the `name` attribute for `Task` is certainly an interesting way to go, and I applaud the choice! What additional considerations would that choice entail when writing a method like `destroy`?